code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -84,6 +84,35 @@
 	glowyeyes = TRUE
 	eyetype = "photie"
 
+/mob/living/simple_animal/otie/redspace
```

Comment:
Although if we're doing redspace oties, but they are called just 'red' oties, should we not just have the file path be `/mob/living/simple_animal/otie/red`? Again, not a deal breaker. Depends what context you want these seen.